ArduPlane/quadplane.cpp

```diff
@@ -1606,7 +1606,7 @@ void SLT_Transition::update()
             // is needed to maintain good control in forward
             // transitions
             quadplane.attitude_control->reset_yaw_target_and_rate();
-            quadplane.attitude_control->rate_bf_yaw_target(quadplane.ahrs.get_gyro().z);
+            quadplane.attitude_control->rate_bf_yaw_target(quadplane.ahrs.get_gyro().z * DEGX100);
```

Comment:
`degrees() * 100.0` if much more commonly used, and easier to understand I think.

Reply:
Got it, better advice.